File: 0334-increasing-triplet-subsequence/0334-increasing-triplet-subsequence.kt
 class Solution {
        fun increasingTriplet(nums: IntArray): Boolean {
            var first = Int.MAX_VALUE
            var sec = Int.MAX_VALUE
            for (i in nums)
            {
                if (i<=first) first = i
                else if (i<=sec) sec = i
                else return true
            }
            return false
        }
    }

File: 0334-increasing-triplet-subsequence/0334-increasing-triplet-subsequence.cpp
class Solution {
public:
    bool increasingTriplet(vector<int>& nums) {
        int first = INT_MAX, sec = INT_MAX;
        for(int i: nums){
            if(i<=first) first = i;
            else if(i<=sec) sec = i;
            else return true;
        }
        return false;
    }
};